### PR TITLE
54 move to red hat publisher

### PR DESCRIPTION
**Pull request by apupier**

### CODE REVIEW

**Comment by apupier** [comment]
using a specifiv branch to test job here https://github.com/camel-tooling/vscode-wsdl2rest/tree/54-specificbranchToTestJob
we can see result with this build https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/VS%20Code/job/vscode-vscode-wsdl2rest-release/13/

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# How to release
+
+- Create a tag and push it
+- Start build [on Jenkins CI](https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/VS%20Code/job/vscode-wsdl2rest-release/) with _publishToMarketPlace_ parameter checked
+- When the build is waiting on step _Publish to Marketplace_, go to the console log of the build and click "Approve"
```
Comment (lhein) [inline Contributing.md]: a manual step in an automated build? Is there no parameter to autoanswer with Approve? Is there some reason for having that manual intervention?

Comment (apupier) [reply Contributing.md]: I followed conventions of other vs code extensions.
cc @jmaury @gorkem might better know th exact purpose.

i think that it allows to test a last time manually the built vsix

Comment (apupier) [reply Contributing.md]: Fred Bricon answered on chat:
"we usually smoke test the vsix one last time, before approving the final release. We had to abort a bunch of times after finding issues"

Comment (lhein) [reply Contributing.md]: Ok. That clarifies the question and also instructs us to do it the same way.

Comment (lhein) [reply Contributing.md]: Maybe the instructions should reflect what Fred said.

**Pull request merged**